cmake/configs/base.cmake

```diff
@@ -60,6 +60,19 @@ config_bool(
     DEFAULT OFF
 )
 
```

Comment:
np + not related to this PR (realised while reading the whole file for some context): line 41 `aligment` -> `alignment`

Reply:
Good find! Fixed :)